.github/workflows/ci.yaml

```diff
@@ -213,6 +213,69 @@ jobs:
         #  testlib modules of gax
         run: mvn package clirr:check -DskipTests -Dfmt.skip
 
+  build-java8-showcase:
+    name: "build(8) for showcase"
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          java-version: 11
+          distribution: temurin
+          cache: maven
+      - name: Install all modules using Java 11
+        shell: bash
+        run: |
+          mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip -Dfmt.skip
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: temurin
+      - run: java -version
+      - name: Showcase golden tests
+        working-directory: java-showcase
+        run: |
+            mvn test \
+            -P enable-golden-tests \
+            --batch-mode \
+            --no-transfer-progress
+      - name: Parse showcase version
+        working-directory: java-showcase/gapic-showcase
+        run: echo "SHOWCASE_VERSION=$(mvn help:evaluate -Dexpression=gapic-showcase.version -q -DforceStdout)" >> "$GITHUB_ENV"
+      - name: Install showcase server
```

Comment:
Downloading showcase server should ideally only happen once, but I guess that may require this to be merged with the regular showcase tests. Is there a way for two CI jobs to share the same set up?

Reply:
Hm true, from looking at the logs, I think this step gets repeated even when we have a matrix. Haven't tried this out yet but we could have a separate job that does the download once and caches the showcase path. All the other showcase jobs would be dependent on this job. wdyt? 

However, this may lead to the "download" job showing on the CI.

Reply:
> I think this step gets repeated even when we have a matrix

That's a good point, I think we can leave it as it is.